DEPS

```diff
@@ -1,26 +1,30 @@
 use_relative_paths = True
 
 deps = {
-  "vendor/requests": "https://github.com/kennethreitz/requests@e4d59bedfd3c7f4f254f4f5d036587bcd8152458",
+  "components/playlist/browser/resources/playlist-component": {
+    "url": "https://github.com/brave/playlist-component.git@5434730bf7342f1ba5c057f1640882bb38604a85",
+    "condition": "not checkout_ios",
```

Comment:
Is this necessary? It will cause problems if you use the same checkout for mac and ios

Reply:
Yes, but that does not preclude us from adding this dep anyway